core/che-core-api-model/src/main/java/org/eclipse/che/api/core/model/factory/Button.java

```diff
@@ -0,0 +1,33 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.api.core.model.factory;
+
+/**
+ * Defines factory button.
+ *
+ * @author Anton Korneta
+ */
+public interface Button {
+
+    enum ButtonType {
+        logo, nologo
```

Comment:
1. I think just `Type` is better, because it's already `Button.Type` and `Button.ButtonType` looks weird for me.
2. Why lowercase? It doesn't fit our codestyle, change it please.